tests/unit/expiry/ExpiryDateStrategyFactory.unitTest.ts

```diff
@@ -40,71 +47,64 @@ describe("ExpiryDateStrategyFactory", () => {
             invalidConfig
           );
         } catch (error) {
-          expect.assertions(1);
           expect(error).toEqual(new Error(ERRORS.ExpiryConfigMissing));
         }
       });
     });
 
     describe("when valid test type id is passed and more than one strategy is returned", () => {
-      it("should throw error", () => {
+      it("should throw an appropriate error", () => {
+        expect.assertions(1);
         try {
+          //  @ts-ignore
+          jest.spyOn(ExpiryDateStrategyFactory, 'getStrategyMapping').mockReturnValue(
+            //  @ts-ignore
+            ExpiryDateStrategyFactory.getStrategyMapping(
+              VEHICLE_TYPE.PSV,
+              duplicatedConfig
```

Comment:
you can return 
 ```
[
                {
                isHistoryRequired: null,
                isRegistrationRequired: null,
                testTypes: ["142", "146", "175", "177"],
                strategy: "PsvDefaultExpiryStrategy"
                },
                {
                isHistoryRequired: false,
                isRegistrationRequired: false,
                testTypes: ["1", "3", "4", "7", "8", "10", "14", "18", "21", "27", "28", "93"],
                strategy: "PsvDefaultExpiryStrategy"
                },
                {
                isHistoryRequired: true,
                isRegistrationRequired: null,
                testTypes: ["1", "3", "4", "7", "8", "10", "14", "18", "21", "27", "28", "93"],
                strategy: "PsvMostRecentExpiryStrategy"
                },
                {
                isHistoryRequired: null,
                isRegistrationRequired: true,
                testTypes: ["1", "3", "4", "7", "8", "10", "14", "18", "21", "27", "28", "93"],
                strategy: "PsvRegistrationAnniversaryStrategy"
                }
            ]
```
instead of calling the function again.

Reply:
Done

Reply:
Nice